assignments/004-iteration-and-fastp/script.sh

```diff
@@ -3,7 +3,7 @@
 # this assumes that you created an environment
 # called fastp in class on Tuesday.  If you accidentally
 # installed fastp into base, then you don't need this line.
-conda activate fastp
+source activate fastp
```

Comment:
This was changed to correct this error where "conda activate" was not being recognized:

usage: conda [-h] [--no-plugins] [-V] COMMAND ...
conda: error: argument COMMAND: invalid choice: 'activate' (choose from 'clean', 'compare', 'config', 'create', 'info', 'init', 'install', 'list', 'notices', 'package', 'remove', 'uninstall',
 'rename', 'run', 'search', 'update', 'upgrade', 'build', 'content-trust', 'convert', 'debug', 'develop', 'doctor', 'index', 'inspect', 'metapackage', 'render', 'skeleton', 'env', 'verify', '
token', 'pack', 'repo', 'server')
./assignments/004-iteration-and-fastp/script.sh: line 25: fastp: command not found
./assignments/004-iteration-and-fastp/script.sh: line 25: fastp: command not found
./assignments/004-iteration-and-fastp/script.sh: line 25: fastp: command not found
./assignments/004-iteration-and-fastp/script.sh: line 25: fastp: command not found
./assignments/004-iteration-and-fastp/script.sh: line 25: fastp: command not found
./assignments/004-iteration-and-fastp/script.sh: line 25: fastp: command not found
./assignments/004-iteration-and-fastp/script.sh: line 25: fastp: command not found
./assignments/004-iteration-and-fastp/script.sh: line 25: fastp: command not found
./assignments/004-iteration-and-fastp/script.sh: line 25: fastp: command not found
./assignments/004-iteration-and-fastp/script.sh: line 25: fastp: command not found
./assignments/004-iteration-and-fastp/script.sh: line 25: fastp: command not found
./assignments/004-iteration-and-fastp/script.sh: line 25: fastp: command not found
./assignments/004-iteration-and-fastp/script.sh: line 25: fastp: command not found
./assignments/004-iteration-and-fastp/script.sh: line 25: fastp: command not found
./assignments/004-iteration-and-fastp/script.sh: line 25: fastp: command not found
./assignments/004-iteration-and-fastp/script.sh: line 25: fastp: command not found

I also had to run the script with "source assignments/004-iteration-and-fastp/script.sh" after this change